### ISSUE TITLE
SVA-Finder: Specify special cases for the Bundesbeauftragten

### ISSUE THREAD

**Issue by mal-tee** [datenanfragen/website#532]
e.g. telecommunication companies

**Comment by baltpeter** [comment]
The BfDI is responsible in particular for ([1](https://stiftungdatenschutz.org/fileadmin/Redaktion/Dossiers_Infoplattform/2020-07-Update/DSGVO-Praxis__Aufsichtsbehoerden_Zustaendigkeiten_Juni2020.pdf)):

* öffentliche Stellen des Bundes
* Telekommunikations- und Postdienstunternehmen
* private Unternehmen, die unter das Sicherheitsüberprüfungsgesetz fallen
* Jobcenter

**Comment by baltpeter** [comment]
I'll put this on hold until datenanfragen/data#1126 is done as we'll also need to account for the Rundfunkdatenschutzbeauftragten in the SvaFinder.

**Comment by baltpeter** [comment]
See also: https://www.bfdi.bund.de/DE/Service/Kontakt/Kontaktfinder/kontaktfinder_node.html

**Comment by moriarty-jim** [comment]
I'm preparing to work on this when datenanfragen/data#1126 is merged. If I understand your code structure correctly, changes need to be made to:

`src/Components/SvaFinder.js` something like this:

```javascript
const STEPS = {
  country: {
    de: {
      // ...
      rundfunk: {
        'rundfunk-dw': 'derfdw',
        // ... (add all broadcasters)
      },
      // ...
    },
    // ...
  },
};
// ...
const SVAS = {
  derfdw: 'Rundfunkdatenschutzbeauftragter Deutsche Welle',
  // ... (add all Rundfunkdatenschutzbeauftragte)
}
```

and all the translation files in `src/i18n/` for languages in use (de, en, fr, hr, pt, es):

```
{
...
    "sva-finder": {
        "rundfunk": "Öffentlich-rechtlicher Rundfunk",
        "rundfunk-q": "Auf welche öffentlich-rechtliche Rundfunkanstalt bezieht sich Dein Anliegen?",
        "rundfunk-dw": "Deutsche Welle",
        ...
     }
...
}
```
Anywhere else? Or am I completely on the wrong track?

**Comment by baltpeter** [comment]
> I'm preparing to work on this when datenanfragen/data#1371 is merged. 

Awesome! I'll assign you in that case.

> Anywhere else? Or am I completely on the wrong track?

No, that should be it. Feel free to ask if anything's unclear but looks like you've already figured it out.

**Comment by moriarty-jim** [comment]
German, English and French translations for this task are no problem for me. What should I do with the other languages (hr, pt, es)? Placeholders?

Specifically these two phrases:
| German  | English | French | 
| ------------- | ------------- | ------------- |
| Öffentlich-rechtlicher Rundfunk  | Public service broadcasters  | Service public de radiodiffusion | 
| Auf welche öffentlich-rechtliche Rundfunkanstalt bezieht sich Dein Anliegen?  | Which public service broadcaster does your request concern?  | Quel radiodiffuseur de service public ta demande concerne-t-elle ? |

- [ ] Croatian translation
- [ ] Portuguese translation
- [ ] Spanish translation

Also, I wouldn't translate the broadcasters names (e.g. "Norddeutscher Rundfunk (NDR)"). What do you think?

**Comment by mal-tee** [comment]
We are using weblate and have some great people keeping the strings updated. That means we only need to specify english as a fallback and all other languages might trickle in over time, when our awesome translators on weblate had the time.

That means for you: As long as you provide Englisch you are fine, everything more is great, but you don't have to cover everything. :)

**Comment by baltpeter** [comment]
Yep, what @mal-tee said. And I agree with not translating the names.

**Comment by moriarty-jim** [comment]
> We are using Weblate and have some great people keeping the strings updated. That means we only need to specify English as a fallback and all other languages might trickle in over time, when our awesome translators on Weblate had the time.
> 
> That means for you: As long as you provide English you are fine, everything more is great, but you don't have to cover everything. :)

I originally just filled in the German "translations" and got errors during testing of the other language sites. I somehow didn't consider that English (and not German) is the default fallback, which - in hindsight - makes perfect sense, of course. That's why I somehow thought that there is no fallback and all translations are necessary. Thanks for clarifying :)

**Comment by baltpeter** [comment]
Either is fine. It should be pretty obvious not to translate them, especially if the English one is already untranslated.
